community/kernel/src/main/java/org/neo4j/kernel/api/cursor/RelationshipCursor.java

```diff
@@ -25,8 +25,53 @@
  * Cursor for iterating over a set of relationships.
  */
 public interface RelationshipCursor
-    extends Cursor
+        extends Cursor
 {
+    RelationshipCursor EMPTY = new RelationshipCursor()
```

Comment:
Same here as about the `PropertyCursor` comment. WDYT? Imho makes things more explicit